j2cl-tasks/src/main/java/com/vertispan/j2cl/build/provided/BytecodeTask.java

```diff
@@ -119,4 +152,23 @@ public Task resolve(Project project, Config config) {
     protected File getGeneratedClassesDir(TaskContext context) {
         return context.outputPath().toFile();
     }
+
+    private Set<String> maybeAddInReactorAptProcessor(List<Input> reactorProcessors, Set<String> processors) {
+        if(processors.isEmpty()) {
+            return Collections.emptySet();
+        }
+        Set<String> existingProcessors = new HashSet<>(processors);
+        reactorProcessors.stream().map(input -> input.filter(APT_PROCESSOR))
+                .forEach(input -> input.getFilesAndHashes().forEach(file -> {
+            try (BufferedReader reader = new BufferedReader(new InputStreamReader(new FileInputStream(file.getAbsolutePath().toFile())))) {
+                String line;
+                while ((line = reader.readLine()) != null) {
+                    existingProcessors.add(line);
+                }
```

Comment:
I think this can be written more briefly as this?
```suggestion
            try (Stream<String> lines = Files.lines(file.getAbsolutePath())){
                lines.forEach(existingProcessors::add);
```

Reply:
agree, old school :)